express-notes-api/index.js

```diff
@@ -0,0 +1,109 @@
+const express = require('express');
+const fs = require('fs');
+const app = express();
+let data = null;
+fs.readFile('./data.json', 'utf-8', (err, dataJSON) => {
+  if (err) {
+    process.exit(1);
+  }
```

Comment:
Not a requirement for the exercise, but if it fails to read the file on start-up, you may want to print out the error message.